serverless-ami-backup.py

```diff
@@ -136,6 +136,11 @@ def amiBakerBot():
                                                 # ToDo: Not able to get only the additional disk in device mappings
                                                 # BlockDeviceMappings = _BlockDeviceMappings,
                                                 NoReboot = True
+            #Wait for AMI to be available. 
+            logger.info("before waiter")
+            waiter = ec2_client.get_waiter('image_available')
+            waiter.wait(Filters=[{'Name':'image-id','Values':[response['ImageId']]}])
+            logger.info("after waiter")
```

Comment:
Can we change the log message. For example,  
This line 
```py
logger.info("after waiter")
``` 
can be changed to something like,
```py
logger.info(f'{{ "image_id" : "{ response["ImageId"] }", "completed_backup": True }}')
```
This will result in a json log file like this,
```json
{ "image_id" : "ami-09d95fab7fff3770c", "completed_backup": true }
```

Reply:
sure. I also modified the before message to be more detailed.